src/aui-form-builder/js/aui-form-builder-pages.js

```diff
@@ -582,6 +609,17 @@ A.FormBuilderPages = A.Base.create('form-builder-pages', A.Base, [], {
         },
 
         /**
+         * Type of the pagination.
+         *
+         * @attribute paginationType
+         * @default 'pagination'
+         * @type {String}
+         */
+        paginationType: {
```

Comment:
It's weird the issue name being "Add attribute mode..." and here you named as paginationType. Could rename paginationType to mode?